### DISCUSSION TITLE
add timestamp like nuclei, or a flag for it.

### DISCUSSION THREAD

**Discussion by gprime31** [projectdiscovery/httpx#487] [Q&A] [answered]
add timestamp for each request like nuclei, or a flag for it.

**Comment by ehsandeep** [comment, marked as answer]
Hey @gprime31, this information is available in JSON output, so will suggest using `json` flag and process as per your need.
We are avoiding new flags for existing information due to the excessive number of flags we have in case of httpx, we will come up with some updates to reduce number of flags + print the information of any desired output field.

**Reply by gprime31** [reply]
Ok, thanks.
